easycv/toolkit/blade/cv_blade_utils.py

```diff
@@ -0,0 +1,248 @@
+import argparse
+import itertools
+import os
+import time
+import timeit
+import logging
+from contextlib import contextmanager
+
+import numpy as np
+import pandas as pd
+import torch
+import torchvision
+
+os.environ['DISC_ENABLE_STITCH'] = os.environ.get('DISC_ENABLE_STITCH', 'true')
+os.environ['DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE'] = os.environ.get('DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE', 'true')
```

Comment:
Those environmental variables are only needed by DISC stitch. It's recommended to be used only when the DISC stitch strategty is perfered.